locales/en/data-classes.ftl

```diff
@@ -141,6 +142,7 @@ spouses-names = Spouses names
 support-tickets = Support tickets
 survey-results = Survey results
 taxation-records = Taxation records
+telecommunications-carrier = Telecommunications carrier
```

Comment:
Shouldn't this be plurals?

Reply:
I just reuse whatever strings HIBP uses: https://haveibeenpwned.com/PwnedWebsites#ApexSMS ([JSON](https://haveibeenpwned.com/api/v3/breach/ApexSMS)):

> _"The incident leaked over 80M records with 23M unique email addresses alongside names, phone numbers and carriers, geographic locations (state and country), genders and IP addresses."_

I don't think we can change the slugified key, but I can change the value to "Telecommunications carriers" if you think thats better for L10n-ing.

Reply:
I think it's better for consistency, if nothing else.

Reply:
Done!